packages/server/database/types/TeamMember.ts

```diff
@@ -42,6 +45,7 @@ export default class TeamMember {
     this.teamId = teamId
     this.email = email
     this.hideAgenda = hideAgenda || false
+    this.hideManageTeam = hideManageTeam || true
```

Comment:
I'm hiding the manage team view by default as I assume the agenda list should take precedent? Manage team view is more of a settings page so I imagine users will use the agenda list more

Reply:
Also, as we default to true if hideManageTeam is undefined, I assume we don't need a migration here?

Reply:
Yes, I think hiding manage team view by default is a good choice